### DISCUSSION TITLE
Yang2017 model infeasible event + huge capacity loss

### DISCUSSION THREAD

**Discussion by sklein-ise** [pybamm-team/PyBaMM#1567] [Q&A] [answered]
Hello,

I have been trying to use the Yang2017 model. I tried to use it with the experiment class to reproduce aging with the cited cycling procedure from the Yang 2017 paper:

>The cell was put into cycling test using an Arbin BT-2000 system (Arbin Instruments, TX) at room temperature. In each cycle, the cell was charged using a constant current (CC) step at the rate of 1C to a cut-off voltage of 4.2 V, followed by a constant voltage (CV) step at 4.2 V until the charge current dropped below C/10. Resting for 5 min after the charging process, the cell was discharged at a constant current of 2C to 2.8 V, followed by another 5 min rest before moving to the next cycle"

But when I implement it, calculating the Jacobian fails and  I get the event 
"Experiment is infeasible: 'event: Maximum negative particle surface concentration' was triggered during 'Charge at 1C until 4.2V'. The returned solution only contains the first 2 cycles. Try reducing the current, shortening the time interval, or reducing the period"

Also no matter how low I set the discharge and charge current the capacity loss is really big even after a few cycles. Am I doing something wrong with the implementation?

**Jacobian + Event**:

```
model = pybamm.lithium_ion.Yang2017()
parameter_values = pybamm.ParameterValues(chemistry=pybamm.parameter_sets.Yang2017)
experiment = pybamm.Experiment(
    [(
      "Discharge at 2C until 2.8V",
      "Rest for 5 minutes",
      "Charge at 1C until 4.2V",
      "Hold at 4.2V until C/10",
      "Rest for 5 minutes",)] * 5,
)
sim =  pybamm.Simulation(model, experiment = experiment, parameter_values=parameter_values)
solution = sim.solve( solver=pybamm.CasadiSolver(mode="safe"))

```

**capacity loss**:

```
 experiment = pybamm.Experiment(
     [(
       "Discharge at 0.02C until 2.8V (10 minute period)",
       "Rest for 5 minutes",
      "Charge at 0.1C until 4.2V (20 minute period)",
       "Hold at 4.2V until C/10 (10 minute period)",
       "Rest for 5 minutes",)] * 5,
 )
 sim =  pybamm.Simulation(model, experiment = experiment, parameter_values=parameter_values)
 solution = sim.solve( solver=pybamm.CasadiSolver(mode="safe"))

 t_sol = solution["Time [s]"].entries
 d_sol = 1000*solution["Discharge capacity [A.h]"].entries
 fig = plt.figure("Time")
 plt.plot(t_sol, solution["Terminal voltage [V]"](t_sol), '--')
 fig = plt.figure("Capacity")
 plt.plot(d_sol, solution["Terminal voltage [V]"](t_sol),'--', color = 'red')

```

Thanks in advance!

**Comment by valentinsulzer** [comment, marked as answer]
The parameter set for the Yang model was incomplete and not publicly available, so after lots of effort to reverse engineer it we have decided to remove that parameter set
